tests/app/legacy/v2/notifications/test_rest.py

```diff
@@ -250,10 +323,9 @@ async def test_v2_sms_returns_400_with_invalid_personalisation(
         response = client.post(self.sms_route, json=sms_request_data)
 
         assert response.status_code == status.HTTP_400_BAD_REQUEST
-
-        response_errors = response.json()['errors']
-        assert len(response_errors) == 1
-        assert 'Missing personalisation:' in response_errors[0]['message']
+        response_json = response.json()
+        assert len(response_json['errors']) == 1
+        assert response_json['errors'][0]['message'] == 'Missing personalisation: content'
```

Comment:
Question:
These error responses were combed over by @ChrisJohnson-CDJ  and @EvanParish . Is this change correct to remain consistent with notification-api's v2 response?